.github/workflows/ci.yml

```
@@ -39,6 +39,8 @@ jobs:
       run: pip install -e . -v
     - name: fetch metadata
       id: fetch_metadata
+      env:
+        GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Let's call this 
```suggestion
        GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
```

(otherwise people won't know where the token on the repo comes from)

---

@unkcpz Can you please add the `GITHUB_API_TOKEN` secret for codespaces in this repo?